compiler/parser/chapel.lex

```diff
@@ -815,16 +809,20 @@ static int processBlockComment(yyscan_t scanner) {
 
       location = wholeComment.find("\\x09");
     }
+    if(!badComment)
+      yyLval->pch = astr(wholeComment.c_str());
+    else {
+      
+      ParserContext context(scanner);
 
-    yyLval->pch = astr(wholeComment.c_str());
-
+      fprintf(stderr, "Warning:%d:chpldoc comment not closed, ignoring comment:%s\n",
```

Comment:
Sorry, this is nearly perfect!  Could you add a space between the line number and the start of the error message?  So it will be "Warning:%d: chpldoc" instead of "Warning:%d:chpldoc".

Be sure to update the .good one more time, and run start_test over that future to make sure it works!  Then if it says the future passes, you should remove the .bad and .future file.  I will run testing just to be certain nothing broke, but I suspect this will then be good to merge!